src/app/content/highlights/components/CardWrapper.tsx

```diff
@@ -98,6 +118,7 @@ const Wrapper = ({highlights, className, container, highlighter}: WrapperProps)
           onHeightChange={(ref: React.RefObject<HTMLElement>) => onHeightChange(highlight.id, ref)}
           zIndex={highlights.length - index}
           shouldFocusCard={focusThisCard}
+          isHidden={checkIfHiddenByCollapsedAncestor(highlight)}
```

Comment:
i think in the existing logic, the card position will end up as like 0 or undefined or something, you could check for that instead of making a new thing

Reply:
@TomWoodward 
with the existing logic the card offset is normally calculated to make the cards appear normally (not jump from the top or anything) when become visible. So it'd say that `isHidden` is still needed.

Reply:
This actually might need a change. I've just found a bug with wrong offset adjusting.